tests/test_analysis/test_closest_atom.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import print_function
+import unittest
+from pytraj import *
+import pytraj as pt
+from utils import fn
+from pytraj.testing import aa_eq
+import numpy as np
+
+class Test(unittest.TestCase):
+	# tests to see if function can find an atom if we search with a coordinate
+	# very close to the atom's xyz coordinates
+	def test_0(self):
+		traj = pt.iterload(fn('Tc5b.x'), fn('Tc5b.top'))
+		frame = traj[0]
+		
+		atom_to_find = 1
+		coords_x, coords_y, coords_z = frame.atom(atom_to_find)
+		close_idx = pt.closest_atom(traj.top, traj[0], (coords_x + 0.01, coords_y + 0.01 ,coords_z + 0.01))
+		aa_eq(close_idx, atom_to_find)
```

Comment:
please use yapf to reformat your code. It seems you're using 8 whitespaces for indentation (should be only 4).